e2e/tests/text/adding-text/index.spec.ts

```diff
@@ -112,7 +112,7 @@ for (const splitByGrapheme of [true, false]) {
       // and there is no style on the part of text that follows, but there is visible text.
       await expect(await canvasUtil.screenshot()).toMatchSnapshot({
         name: `6-after-adding-a-newline-splitByGrapheme-${splitByGrapheme}.png`,
-        maxDiffPixelRatio: 0.01,
+        maxDiffPixelRatio: 0.009,
```

Comment:
```
await expect(page).toHaveScreenshot(`${name}.png`, {
          clip: {
            x,
            y,
            width,
            height,
          },
          maxDiffPixels, // insteadof percentage
        });
```

Reply:
where is this clip in the documentation? i looked for something like this.

Reply:
oh ok is an entirely different way to do visual comparison. the locator can't be clipped.

Reply:
but the image can, right?